olive/passes/onnx/perf_tuning.py

```diff
@@ -42,15 +43,8 @@ def generate_tuning_combos(model, config):
 
 def tune_onnx_model(model, config):
     latency_user_config = {}
-    for eval_config in [
-        "data_dir",
-        "dataloader_func",
-        "batch_size",
-        "input_names",
-        "input_shapes",
-        "input_types",
-        "device",
-    ]:
+    # which should be the same as the config in the metric
+    for eval_config in get_properties_from_metric_type(MetricType.LATENCY):
```

Comment:
Not a breaking issue but can we only add it to `latency_user_config` if it is also in `config`? Right now, this makes the values for all parameters not in `config` as `None` because of the `.get` but this might be problematic in the future if the actual default value is not `None`.

Reply:
Good cache. I think it reasonable, will update it.